translation.rst

```diff
@@ -160,27 +157,121 @@ Now, if the language of the user's locale is French (e.g. ``fr_FR`` or ``fr_BE``
 the message will be translated into ``J'aime Symfony``. You can also translate
 the message inside your :ref:`templates <translation-tags>`.
 
+.. _translation-real-vs-keyword-messages:
+
+.. sidebar:: Using Real or Keyword Messages
```

Comment:
This sidebar is so long ... that it may be better displayed as a regular section. First because of how sidebars are rendered in the public website ... and second, for us, because sidebars require certain indentation which complicates changes and maintenance.